test/server/lib/db/query.js

```diff
@@ -9,7 +9,7 @@ const stubs = {
 	one: sinon.stub().resolves(),
 	manyOrNone: sinon.stub().resolves()
 };
-const fakeQuery = proxyquire('../../../../src/lib/db/query', {
+let fakeQuery = proxyquire('../../../../src/lib/db/query', {
```

Comment:
Keep it `const` - see other comment about doing the `proxyquire` mock injections in one place